tests/unit/storage/createInMemoryStorage.ts

```diff
@@ -219,18 +218,18 @@ registerSuite({
 	},
 
 	'isUpdate': {
-		'Should return `isUpdate=false` when item doesn\'t exist.'(this: any) {
-			const { dfd, storage, data } = getStorageAndDfd(this);
-			storage.isUpdate(data[0]).then(function(result) {
-				assert.isFalse(result.isUpdate);
-			}).then(dfd.resolve);
-		},
-		'Should return `isUpdate=true` when item exists.'(this: any) {
-			const { dfd, storage, data } = getStorageAndDfd(this);
-			storage.add(data);
-			storage.isUpdate(data[0]).then(function(result) {
-				assert.isTrue(result.isUpdate);
-			}).then(dfd.resolve);
-		}
+		// 'Should return `isUpdate=false` when item doesn\'t exist.'(this: any) {
```

Comment:
Do we want to leave commented out code, even in tests? If there is a reason then probably should add a comment to that affect.